server/test/services/export/PdfExporterTest.java

```diff
@@ -212,6 +215,160 @@ public void exportApplication_eligibility() throws IOException, DocumentExceptio
     assertThat(textFromPDFTwo).contains("Meets eligibility");
   }
 
+  @Test
+  public void exportProgram_hasMainProgramInfo() throws IOException {
+    PdfExporterService service = instanceOf(PdfExporterService.class);
+    ProgramDefinition programDef = fakeProgram.getProgramDefinition();
+
+    PdfExporter.InMemoryPdf result =
+        service.generateProgramPreviewPdf(programDef, getFakeQuestionDefinitions());
+
+    List<String> linesFromPdf = getPdfLines(result);
+    assertThat(linesFromPdf.get(0)).isEqualTo(programDef.localizedName().getDefault());
```

Comment:
```suggestion
    assertThat(linesFromPdf).isNotEmpty();
    assertThat(linesFromPdf.get(0)).isEqualTo(programDef.localizedName().getDefault());
```

suggestion: Assert the list is not empty before calling for the first element

Reply:
Updated.